### DISCUSSION TITLE
Code Annotation in OJS Cell not working.

### DISCUSSION THREAD

**Discussion by thegilpatrick** [quarto-dev/quarto-cli#6140] [Q&A]
### Description

````qmd
---
title: OJS Code Annotation
code-annotations: below
format: html
---

```{ojs}

variable = 5; // <1>

```
1.  Does this work?
````
Below is my output in Quarto 1.4.176

![code-annotation-ojs](https://github.com/quarto-dev/quarto-cli/assets/61705820/46bdc55e-eca3-40a5-af08-c8a911f0e303)

**Comment by cscheid** [comment]
No, code annotation isn't supported for OJS. For unfortunate, complicated reasons, it's unlikely to work in the near future. You should still be able to do it in two separate steps, with a javascript code block that contains the annotations, and an OJS block with `echo: false` that contains the executable code.
